### PR TITLE
[bitnami/node] Add tests and publish using VIB

### PR DESCRIPTION
**Pull request by joancafom**
### Description of the change

The main objective of this PR is to publish our Bitnami Node Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding Cypress tests.
- Increasing the existing test coverage of the asset by adding GOSS tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/joancafom/charts/pull/64

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by rafariossaa — APPROVED** [review]
LGTM

File: .vib/node/cypress/cypress.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (recena) [inline .vib/node/cypress/cypress.json:1]: Why do we need functional tests for a runtime as Node?

**Comment by recena** [comment]
By the way, this PR description should follow [the template](https://github.com/bitnami/charts/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

File: .vib/node/cypress/cypress.json (hunk 1 above)
Comment (recena) [reply .vib/node/cypress/cypress.json:1]: Are we delivering a demo app or similar?

Comment (joancafom) [reply .vib/node/cypress/cypress.json:1]: Yes, the chart itself has params to clone an app from a repo and deploy it 😁 

https://github.com/bitnami/charts/blob/6d6c64ff11b4953b89f73218bc4b907345247cb1/bitnami/node/values.yaml#L446-L452

File: .vib/node/cypress/cypress/support/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
[the remark this answers is not in the document]
Comment (joancafom) [reply .vib/node/cypress/cypress/support/index.js]: You are right. If I am not wrong, this file used to be mandatory, but it seems the current version of Cypress does not require it. I have updated the PR, thanks!

**Review by FraPazGal — APPROVED** [review]
Thanks for the PR!

**Pull request merged**